HelpSource/Classes/Char.schelp

```diff
@@ -108,3 +108,47 @@ code::
 	[i,a = i.asAscii,a.isAlphaNum,a.isPrint,a.isPunct,a.isControl].postln;
 });
 ::
+
+method::isLower
+
+returns:: true if the char is a lowercase letter.
+
+method::isUpper
+
+returns:: true if the char is a uppercase letter.
+
+method::isPathSeparator
+
+returns:: true if the char is a platform specific path separator.
```

Comment:
should probably be
`returns:: true if the char is the platform specific path separator.`
(since it returns false for platform-specific path seperators for other platforms ;) )